Add wheel builds (#12096)
rapidsai/cudf: python/cudf/CMakeLists.txt

option(FIND_CUDF_CPP "Search for existing CUDF C++ installations before defaulting to local files"
       OFF
)

> option(CUDF_BUILD_WHEELS "Whether we're building a wheel for pypi" OFF)

Reviewer: Also suggested on https://github.com/rapidsai/rmm/pull/1148. Maybe check that PR for any other discrepancies.
```suggestion
option(CUDF_BUILD_WHEELS "Whether this build is generating a Python wheel." OFF)
```

Author: Good call, I'll fix this in all the others too.
